src/store.ts

```diff
@@ -167,6 +168,12 @@ export function useWorkflow (): UseWorkflow {
       setGlobal(parameters)
       setFiles(newUsedFiles)
     },
+    clearNodes () {
+      const blankNodes = removeAllItems(nodes)
+      const blankGlobals = emptyParams()
+      setNodes(blankNodes)
+      setGlobal(blankGlobals)
```

Comment:
The files are not cleared yet
```suggestion
      setGlobal(blankGlobals)
      const blankFiles: IFiles = {}
      setFiles(blankFiles)
```

Reply:
ops!